src/helpers/utils.ts

```diff
@@ -117,3 +127,43 @@ export function filterProposals(space, proposal, tab) {
 
   return false;
 }
+
+export function scrollEndMonitor(fn) {
+  let canRunAgain = true;
+
+  const el = document.getElementById('endofpage');
+  const elementWatcher = scrollMonitor.create(el);
+  elementWatcher.enterViewport(() => {
+    if (canRunAgain) {
+      canRunAgain = false;
+      fn();
+
+      setTimeout(function () {
+        canRunAgain = true;
+      }, 100);
+    }
+  });
+}
+
+export const formatAmount = (amount, maxDecimals) => {
```

Comment:
This doesn't seem to be used, you can keep it if you think you'll need it later on.

Reply:
I think I will, so I'm going to keep it 👍🏻